### PR TITLE
consistency check done between readme and index.js

### PR DESCRIPTION
**Pull request by Archulan**
@strdr4605 , I have done the consistency check between readme file and index.js. Please consider my pull request.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -155,21 +157,6 @@ function randomCase(input) {
...
 }
 
-/**
- * Converts matched letters to a specified case
- * @param {string} input - string to convert
- * @param {(string | RegExp)} regex - use to match letters you want to convert
- * @param {function} caseReplacer - a function that changes the letter's case
- * @return {string} given input with match letters to the  case type determined
- */
-function convertToSpecificCase(input, regex, caseReplacer) {
```
Comment (strdr4605) [inline index.js]: Why is this function removed?! It is used at https://github.com/strdr4605/mockingcase/blob/master/src/mockingcase.js#L54

Comment (Archulan) [reply index.js]: Very sorry @strdr4605 . I will add it and give you a pull request. Is that the only error?

Comment (strdr4605) [reply index.js]: Till now no, but I will check again after the update. Please remove `.idea` folder, and `rebase` your branch on top of the main master branch

Comment (Archulan) [reply index.js]: Okay. I will rebase my branch and give you a PR within a day.

**Comment by strdr4605** [comment]
Close in favor of #80

**Pull request closed without merging**